SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -192,235 +192,235 @@ bool CreatePlaceholders(Worker_SnapshotOutputStream* OutputStream)
 
 // This function is not in use.
 // Set up classes needed for Startup Actor creation
-void SetupStartupActorCreation(USpatialNetDriver*& NetDriver, USpatialNetConnection*& NetConnection, USpatialPackageMapClient*& PackageMap, USpatialClassInfoManager*& ClassInfoManager, UWorld* World)
-{
-	NetDriver = NewObject<USpatialNetDriver>();
-	NetDriver->ChannelClasses[CHTYPE_Actor] = USpatialActorChannel::StaticClass();
-	NetDriver->GuidCache = MakeShareable(new FSpatialNetGUIDCache(NetDriver));
-	NetDriver->World = World;
-
-	ClassInfoManager = NewObject<USpatialClassInfoManager>();
-	ClassInfoManager->Init(NetDriver);
-
-	NetDriver->ClassInfoManager = ClassInfoManager;
-
-	NetConnection = NewObject<USpatialNetConnection>();
-	NetConnection->Driver = NetDriver;
-	NetConnection->State = USOCK_Closed;
-
-	PackageMap = NewObject<USpatialPackageMapClient>();
-	PackageMap->Initialize(NetConnection, NetDriver->GuidCache);
-
-	NetConnection->PackageMap = PackageMap;
-	NetDriver->PackageMap = PackageMap;
-}
+//void SetupStartupActorCreation(USpatialNetDriver*& NetDriver, USpatialNetConnection*& NetConnection, USpatialPackageMapClient*& PackageMap, USpatialClassInfoManager*& ClassInfoManager, UWorld* World)
```

Comment:
What's going on with all this commented out?

Reply:
this is for reference since it was hard to do this initially

Reply:
Though we can delete it I guess. Mike also wants to delete it.

Reply:
+1 to deleting it